Dockerfile

```diff
@@ -0,0 +1,68 @@
+# syntax=docker/dockerfile:1
+ARG PYTHON_VERSION=3.10
+
+##################################################
+#                   Python                       #
+##################################################
+FROM python:${PYTHON_VERSION}-slim-buster AS signals_python
```

Comment:
I think you can omit AS signals_python`, as we are not leveraging multi-stage builds.

Reply:
True, I added it because I was planning to replace the other amsterdam registry images as well, but as it turns out I'm not going to right now. I'll remove it for the time being.